### PR TITLE
Update no-food-is-fully-vegan.md

### PR DESCRIPTION
**Pull request by DrTeo**
Unpacked argument into 3 forms
Resonses to each argument
Doctrine of double effect
Distinction of acting with foreknowledge
Defining 'possible and practicable'.

### CODE REVIEW

**Comment by DrTeo** [comment]
Got quite long...

In some ways it feels a bit rushed as I cut it back from growing even further.

Needs a fresh pair of eyes (mine are asleep!)

File: content/english/anti-vegan/no-food-is-fully-vegan.md [hunk 1]
```diff
@@ -11,33 +11,104 @@ weight: 5
...
 
-> That's completely correct, it is almost impossible to avoid being complicit in suffering with our current food chain. Veganism is a way of living which seeks to exclude, as far as is possible and practicable, all forms of exploitation of, and cruelty to, animals [...]. Raising an animal from forcible impregnation to their untimely death merely to satisfy our desire for taste is not comparable to deaths we have no way of avoiding as consumers.  
+> (Argument #1) This argument is an appeal to hypocrisy (*the Tu quoque fallacy*). That I fail to meet my own standards is nothing more than a personal failure and does not invalidate my principles. My failure to pick up a piece of litter that I've dropped does not invalidate my claim that littering is wrong. That I may fall short and in some way be complicit in an animal's death in no way invalidates the moral claim that the exploitation of animals is wrong.
+> 
+> (Argument #2) "It is wrong to inflict unnecessary death on nonhuman animals and it is especially wrong , pro tanto, to do so intentionally. Where one must choose between bringing about death as a side-effect, one should, all else equal, pursue the latter course of action".
```
Comment (Sam-Martin) [inline content/english/anti-vegan/no-food-is-fully-vegan.md]: extra space `especially wrong ,`

File: content/english/anti-vegan/no-food-is-fully-vegan.md [hunk 2]
```diff
@@ -11,33 +11,104 @@ weight: 5
...
+> Argument #1 is a fallacy (the appeal to hypocrisy).
+
+### Argument #2 *Intent & Knowledge*
+
+Vegans must eat and, where there are limited options, follow the path of least harm.
+
+
+>A golden rule of moral philosophy is, "all things being equal you ought to choose the action that does the least harm”.
+
+While it is true that some animals do die in the production and harvesting of crops vegans do not ‘accept these deaths’ without question.  Where a crop farmer fails to *exercise due diligence to minimize the harm* and deaths occur, an obligation falls on vegans *as far as is possible and practicable* to seek alternatives (e.g. moving towards organic vegetables, buying from vegan farms, avoiding mono-cultures etc.)
```
Comment (Sam-Martin) [inline content/english/anti-vegan/no-food-is-fully-vegan.md]: Let's keep the viva link to Vegan Farming here: 
```
[vegan farms(https://www.viva.org.uk/what-we-do/vegan-farming)
```

File: content/english/anti-vegan/no-food-is-fully-vegan.md [hunk 3]
```diff
@@ -11,33 +11,104 @@ weight: 5
...
+
+> ### Doctrine of Double Effect
+> - the nature of the act is itself good, or at least morally neutral;  
+> - the individual intends the good effect and does not intend the bad effect, either as a means to the good or as an end in itself;  
+> - the good effect outweighs the bad effect in circumstances sufficiently grave to justify causing the bad effect, and the individual exercises due diligence to minimize the harm.  
+> 
+> *from* Double-Effect Reasoning: Doing Good and Avoiding Evil *by* T. A. Cavanaugh.
+ 
+While cattle farmer's *intend* to kill the cattle they raise, the crop farmer's intention is primarily to grow crops. Any deaths that may occur during crop harvest are a side-effect of a morally neutral act.
```
Comment (Sam-Martin) [inline content/english/anti-vegan/no-food-is-fully-vegan.md]: I would prefix this sentence with something like `In other words, When cattle farmers...` as I think most people will skip over the doctrine of double effect as "intimidating philosophy words" and look to the next unquoted sentence to repeat it in laymans terms. (I think we should keep the definition to be clear as it's super clear if you take the time to read it, we just need to make 100% sure there is a simplified version that people are drawn to read).

Also `cattle farmers` rather than `cattle farmer's`

File: content/english/anti-vegan/no-food-is-fully-vegan.md [hunk 4]
```diff
@@ -11,33 +11,104 @@ weight: 5
...
+> 
+> *from* Double-Effect Reasoning: Doing Good and Avoiding Evil *by* T. A. Cavanaugh.
+ 
+While cattle farmer's *intend* to kill the cattle they raise, the crop farmer's intention is primarily to grow crops. Any deaths that may occur during crop harvest are a side-effect of a morally neutral act. 
+
+- A counter might follow that 'regardless of intention or acceptance the animals die, so I might as well get my meat.'
+
+This counter argument fails to take into account an individual's foreknowledge of what will occur.
+
+* A cattle farmer has foreknowledge that the cattle will be slaughtered and intends for them to be slaughtered.
```
Comment (Sam-Martin) [inline content/english/anti-vegan/no-food-is-fully-vegan.md]: Can we substitute the word `has foreknowledge` with `knows without a doubt` or just `knows`?

File: content/english/anti-vegan/no-food-is-fully-vegan.md [hunk 5]
```diff
@@ -11,33 +11,104 @@ weight: 5
...
+#### * What does *possible and practicable* mean?
+
+This is not an excuse to have a steak when there isn't a vegan option at the restaurant. Eating a boring salad is a very minor discomfort by comparison with the horrors that animals face in slaughterhouses. 
+
+Nor is it an excuse to not learn those pesky [E-numbers] ({{< ref "../shopping/e-numbers" >}}). The mistreatment of animals does not compare to committing these to memory, or carrying around the TalkVeganToMe App.
+
+
+
+> *Possible and practicable* is NOT a line vegans draw for themselves to mitigate some perceived inconvenience. This would reduce veganism to some form of moral relativism.   
+> Rather, it is a reference to the threshold a ‘rational person’ would set; ‘all other things being equal’.
```
Comment (Sam-Martin) [inline content/english/anti-vegan/no-food-is-fully-vegan.md]: As much as I agree philosophically, I think this teeters too close to the edge of transgressing [mission statement #6](https://github.com/talkvegantome/talkvegan-hugo)

> 6 Not to engage in the gatekeeping of veganism (no-one can take away your V card)

Or at the very least provides a stepping stone for other people to do so. 

Can we do without it?

File: content/english/anti-vegan/no-food-is-fully-vegan.md [hunk 6]
```diff
@@ -11,33 +11,104 @@ weight: 5
...
+
+Where vegans meet or exceed this threshold they act morally (in this regard). 
+
+Where vegans fail to meet this threshold they should acknowledge falling short. It's necessary to recognise that it is nearly impossible to exist without cruelty to animals or people while still participating in a 21st century society, and to forgive ourselves for the things we cannot avoid.
+
+
 
-This is not an excuse to have a steak when there isn't a vegan option at the restaurant. Eating a boring salad is a very minor discomfort in comparison to choosing to endorse the horrors animals face in slaughterhouses. It is only that it is necessary to recognise that it is nearly impossible to without cruelty to animals or people while still participating in a 21st century society, and to forgive ourselves for the things we cannot avoid.
\ No newline at end of file
+> Argument #3 misrepresents the nature of veganism. As with any moral standpoint perfection is not expected. However, there is the expectation of striving towards excellence…
```
Comment (Sam-Martin) [inline content/english/anti-vegan/no-food-is-fully-vegan.md]: Was this a note for yourself or is this intended to be left in?

**Review by Sam-Martin — CHANGES_REQUESTED** [review]

File: content/english/anti-vegan/no-food-is-fully-vegan.md (hunk 3 above)
Comment (DrTeo) [reply content/english/anti-vegan/no-food-is-fully-vegan.md]: In other words a cattle farmer *intends* to kill the cattle, the crop farmer's intention is primarily to grow crops. Any deaths that may occur during crop harvest are a side-effect of a morally neutral act.

File: content/english/anti-vegan/no-food-is-fully-vegan.md (hunk 4 above)
Comment (DrTeo) [reply content/english/anti-vegan/no-food-is-fully-vegan.md]: Left first foreknowledge.

Changed seconds to: 

A cattle farmer knows that the cattle will be slaughtered and intends for them to be slaughtered.

and

A crop farmer knows that some animals (e.g rabbits, birds, rodents) may be killed...

File: content/english/anti-vegan/no-food-is-fully-vegan.md (hunk 6 above)
Comment (DrTeo) [reply content/english/anti-vegan/no-food-is-fully-vegan.md]: Yes and no.   I was asleep at this point.  An ending to the section / Argument #3

How about:

Argument #3 misrepresents the nature of veganism. As with any moral standpoint perfection is not expected. Veganism, and the ideal of a world without exploitation and cruelty, is both logical and possible.

File: content/english/anti-vegan/no-food-is-fully-vegan.md (hunk 3 above)
Comment (Sam-Martin) [reply content/english/anti-vegan/no-food-is-fully-vegan.md]: perfect 👍

File: content/english/anti-vegan/no-food-is-fully-vegan.md (hunk 4 above)
Comment (Sam-Martin) [reply content/english/anti-vegan/no-food-is-fully-vegan.md]: Works for me!

File: content/english/anti-vegan/no-food-is-fully-vegan.md (hunk 6 above)
Comment (Sam-Martin) [reply content/english/anti-vegan/no-food-is-fully-vegan.md]: Rather than adding a caveat at the end can we tweak argument three to represent the messiness of reality? As that misrepresentation is a bit of an issue! (see other comment)

File: content/english/anti-vegan/no-food-is-fully-vegan.md [hunk 7]
```diff
@@ -11,33 +11,103 @@ weight: 5
...
+> Argument #1 is a fallacy (the appeal to hypocrisy).
+
+### Argument #2 *Intent & Knowledge*
+
+Vegans must eat and, where there are limited options, follow the path of least harm.
+
+
+>A golden rule of moral philosophy is, "all things being equal you ought to choose the action that does the least harm”.
+
+While it is true that some animals do die in the production and harvesting of crops vegans do not ‘accept these deaths’ without question.  Where a crop farmer fails to *exercise due diligence to minimize the harm* and deaths occur, an obligation falls on vegans *as far as is possible and practicable* to seek alternatives (e.g. moving towards organic vegetables, buying from [vegan farms] (https://www.viva.org.uk/what-we-do/vegan-farming), avoiding mono-cultures etc.)
```
Comment (Sam-Martin) [inline content/english/anti-vegan/no-food-is-fully-vegan.md]: `[vegan farms] (https://www.viva.org.uk/what-we-do/vegan-farming)` doesn't work as a link, the space needs removing!

File: content/english/anti-vegan/no-food-is-fully-vegan.md [hunk 8]
```diff
@@ -11,33 +11,103 @@ weight: 5
...
+* A cattle farmer knows that the cattle will be slaughtered and intends for them to be slaughtered.
 
-> If someone was driving their car and they accidentally hit a dog, that would not be the same as if they purposefully drove after the dog until they ran them over.  
-> - Ed Winters - 30 Non Vegan Excuses - p72
+* A crop farmer knows that some animals (e.g rabbits, birds, rodents) may be killed during harvest but the intention is only to harvest crops. (The crop farmer would far prefer the animals were not in the field).
+ 
+Consider the following thought experiment:
 
-Intention matters in the morality of your actions. The unintentional deaths of earthworms when tilling, or field mice when harvesting are not _necessary_ for the production of crops, it is not the intention to kill animals when harvesting crops. It is the definitionally the intention when raising animals for slaughter.
+> You are controlling a tram. As long as you hold a lever the tram will continue on its current track.  However, you know there are numerous people on the track ahead who will be killed if the tram continues.
```
Comment (Sam-Martin) [inline content/english/anti-vegan/no-food-is-fully-vegan.md]: Can we remove some of the qualifiers on this? Specifically:
- Numerous
- Very probably

So we would get 

>  You are controlling a tram. As long as you hold a lever the tram will continue on its current track.  However, you *know* there are people on the track ahead who will be killed if the tram continues.  
> If you release the lever the tram will change tracks. You *suspect* there are people on the other track as well.

It conveys the same uncertainty but makes it easier to discern the difference.

File: content/english/anti-vegan/no-food-is-fully-vegan.md [hunk 9]
```diff
@@ -11,33 +11,103 @@ weight: 5
...
 
-Intention matters in the morality of your actions. The unintentional deaths of earthworms when tilling, or field mice when harvesting are not _necessary_ for the production of crops, it is not the intention to kill animals when harvesting crops. It is the definitionally the intention when raising animals for slaughter.
+> You are controlling a tram. As long as you hold a lever the tram will continue on its current track.  However, you know there are numerous people on the track ahead who will be killed if the tram continues.  
+> If you release the lever the tram will change tracks. You suspect there are *very probably* people on the other track as well.
 
-This distinction falls down somewhat when we consider pesticides, animal-based fertilizers, and beekeeping for pollination. This is why some vegans seek out foods from [vegan farms](https://www.viva.org.uk/what-we-do/vegan-farming). Ultimately however each person's 'possible and practicable' is a line they have to draw for themselves and this may not be within the means or motivations of every vegan. It is not necessary to be perfect to be vegan.
+1. Continuing to hold the lever is to kill with both foreknowledge and intent.
```
Comment (Sam-Martin) [inline content/english/anti-vegan/no-food-is-fully-vegan.md]: I think we can just say *knowledge* here.

File: content/english/anti-vegan/no-food-is-fully-vegan.md [hunk 10]
```diff
@@ -11,33 +11,103 @@ weight: 5
...
 
-### Moral Consistency
+2. Releasing the lever is to have the foreknowledge that some people might be killed.
```
Comment (Sam-Martin) [inline content/english/anti-vegan/no-food-is-fully-vegan.md]: I think we can just say *know* here and emphasise *might*.

File: content/english/anti-vegan/no-food-is-fully-vegan.md [hunk 11]
```diff
@@ -11,33 +11,103 @@ weight: 5
...
 
-Trying to trip vegans up by exposing real or imagined inconsistencies is a common avenue of argument against veganism. The reality is that vegans (being human) are typically not perfect. Vegans are subject to the constraints of time, money, knowledge, and availability of resources amongst a million other factors. We quoted the definition of veganism in our quick response but it's so important it bears repeating: 
+Knowingly risking some deaths is morally preferable to knowingly killing with intent.
```
Comment (Sam-Martin) [inline content/english/anti-vegan/no-food-is-fully-vegan.md]: Can we say
> Knowingly risking some deaths is morally preferable to killing with certainty and intent.
I think it's easier to read.

File: content/english/anti-vegan/no-food-is-fully-vegan.md [hunk 12]
```diff
@@ -11,33 +11,103 @@ weight: 5
...
 
-> Veganism is a way of living which seeks to exclude, as far as is possible and practicable, all forms of exploitation of, and cruelty to, animals  for food, clothing or any other purpose.  
+Humans hold a lever which results in the intentional deaths of millions of animals. It's time to let go of the lever; the foreknowledge that some animals will probably die as a result of crop farming is not an argument against veganism.
```
Comment (Sam-Martin) [inline content/english/anti-vegan/no-food-is-fully-vegan.md]: I love this so much, if we ever sell t-shirts we need one that says "let go of the lever".

File: content/english/anti-vegan/no-food-is-fully-vegan.md [hunk 13]
```diff
@@ -11,33 +11,103 @@ weight: 5
...
+
+> Vegans do not accept the deaths of some animals to produce our foods of preference. While vegans have foreknowledge that this may occur it is not the intent and, wherever possible, steps are taken to reduce and mitigate this harm.
```
Comment (Sam-Martin) [inline content/english/anti-vegan/no-food-is-fully-vegan.md]: Let's just say 'While vegans know that this may occur". The fore in foreknowledge isn't doing any useful work for the layman reading this.

File: content/english/anti-vegan/no-food-is-fully-vegan.md [hunk 14]
```diff
@@ -11,33 +11,103 @@ weight: 5
...
+Knowingly risking some deaths is morally preferable to knowingly killing with intent.
 
-> Veganism is a way of living which seeks to exclude, as far as is possible and practicable, all forms of exploitation of, and cruelty to, animals  for food, clothing or any other purpose.  
+Humans hold a lever which results in the intentional deaths of millions of animals. It's time to let go of the lever; the foreknowledge that some animals will probably die as a result of crop farming is not an argument against veganism.
+
+> Vegans do not accept the deaths of some animals to produce our foods of preference. While vegans have foreknowledge that this may occur it is not the intent and, wherever possible, steps are taken to reduce and mitigate this harm. 
+> 
+>  There is considerable moral difference between slaughtering animals with intent and deaths that occur as an unintentional side effect of crop farming, even where we have foreknowledge the deaths might occur.
```
Comment (Sam-Martin) [inline content/english/anti-vegan/no-food-is-fully-vegan.md]: We can get away with `know` again I think.

File: content/english/anti-vegan/no-food-is-fully-vegan.md [hunk 15]
```diff
@@ -11,33 +11,103 @@ weight: 5
...
+
+### Argument #3 *Definition of Veganism*
+
+The definition of veganism is in our Quick Response but it's so important it's worth repeating: 
+
+> Veganism is a way of living which seeks to exclude, as far as is possible and practicable, all forms of exploitation of, and cruelty to, animals for food, clothing or any other purpose.  
 > - [The Vegan Society - Definition of Veganism](https://www.vegansociety.com/go-vegan/definition-veganism)
 
-Each person's 'possible and practicable' is a line they have to draw for themselves. It is not necessary to be perfect to be vegan. 
+The claim that ‘veganism is logically impossible' misrepresents the nature of veganism. As with any moral standpoint perfection is not expected. Rather, vegans strive toward the ideal of a world without exploitation and cruelty. However, it is necessary to recognise that it is nearly impossible to exist without cruelty to animals or people while still participating in a 21st century society, and to forgive ourselves for the things we cannot avoid.
```
Comment (Sam-Martin) [inline content/english/anti-vegan/no-food-is-fully-vegan.md:95]: Perfect, love this too.

File: content/english/anti-vegan/no-food-is-fully-vegan.md [hunk 16]
```diff
@@ -11,33 +11,103 @@ weight: 5
...
-Each person's 'possible and practicable' is a line they have to draw for themselves. It is not necessary to be perfect to be vegan. 
+The claim that ‘veganism is logically impossible' misrepresents the nature of veganism. As with any moral standpoint perfection is not expected. Rather, vegans strive toward the ideal of a world without exploitation and cruelty. However, it is necessary to recognise that it is nearly impossible to exist without cruelty to animals or people while still participating in a 21st century society, and to forgive ourselves for the things we cannot avoid. 
+
+To exist, as far as is possible and practicable*, without exploitation of, or cruelty to, animals is both logical and possible.
+
+#### * What does *possible and practicable* mean?
+
+This is not an excuse to have a steak when there isn't a vegan option at the restaurant. Eating a boring salad is a very minor discomfort by comparison with the horrors that animals face in slaughterhouses. 
+
+Nor is it an excuse to not learn those pesky [E-numbers] ({{< ref "../shopping/e-numbers" >}}). The mistreatment of animals does not compare to committing these to memory, or carrying around the TalkVeganToMe App.
```
Comment (Sam-Martin) [inline content/english/anti-vegan/no-food-is-fully-vegan.md]: the space needs removing in this link too. It should be:
```
[E-numbers]({{< ref "../shopping/e-numbers" >}})
```

File: content/english/anti-vegan/no-food-is-fully-vegan.md [hunk 17]
```diff
@@ -11,33 +11,103 @@ weight: 5
 
 > It's impossible to have completely vegan food. Fertilizers are derived from animal waste and bones, pesticides are used on crops to kill insects, bees are kept in hives to pollinate almonds, worms are killed by tilling fields. All food involves death.
```
Comment (Sam-Martin) [inline content/english/anti-vegan/no-food-is-fully-vegan.md]: Do you mind adding 'portable' to hives while you're here @DrTeo ?

File: content/english/anti-vegan/no-food-is-fully-vegan.md [hunk 18]
```diff
@@ -11,33 +11,103 @@ weight: 5
...
 
-> That's completely correct, it is almost impossible to avoid being complicit in suffering with our current food chain. Veganism is a way of living which seeks to exclude, as far as is possible and practicable, all forms of exploitation of, and cruelty to, animals [...]. Raising an animal from forcible impregnation to their untimely death merely to satisfy our desire for taste is not comparable to deaths we have no way of avoiding as consumers.  
+> (Argument #1) This argument is an appeal to hypocrisy (*the Tu quoque fallacy*). That I fail to meet my own standards is nothing more than a personal failure and does not invalidate my principles. My failure to pick up a piece of litter that I've dropped does not invalidate my claim that littering is wrong. That I may fall short and in some way be complicit in an animal's death in no way invalidates the moral claim that the exploitation of animals is wrong.
+> 
+> (Argument #2) "It is wrong to inflict unnecessary death on nonhuman animals and it is especially wrong, pro tanto, to do so intentionally. Where one must choose between bringing about death as a side-effect, one should, all else equal, pursue the latter course of action".
```
Comment (Sam-Martin) [inline content/english/anti-vegan/no-food-is-fully-vegan.md]: Can we add `[(all other things being equal)]` after pro tanto like Val suggests?

File: content/english/anti-vegan/no-food-is-fully-vegan.md (hunk 7 above)
Comment (DrTeo) [reply content/english/anti-vegan/no-food-is-fully-vegan.md]: Weird. Don't have a space (in my local copy) and it works fine (locally).  re-editing it anyway in the hope it fixes....

File: content/english/anti-vegan/no-food-is-fully-vegan.md (hunk 14 above)
Comment (DrTeo) [reply content/english/anti-vegan/no-food-is-fully-vegan.md]: Think the point is logically stronger using foreknowledge on this occasion.  Not stuck on it though. Will change if you think its a biggie.

File: content/english/anti-vegan/no-food-is-fully-vegan.md (hunk 18 above)
Comment (DrTeo) [reply content/english/anti-vegan/no-food-is-fully-vegan.md]: No.   Well not really... it's doesn't really capture the meaning.

That said I couldn't come up with anything short that did. I've currently ...'d it out (Didn't want to but it either that or a paragraph of explanation.

**Review by Sam-Martin — APPROVED** [review]
